### PR TITLE
Eco news module ver.2

### PR DESCRIPTION
**Pull request by nvolodymyr**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -113,21 +113,22 @@ import { SwiperModule } from 'ngx-swiper-wrapper';
...
 import { InputcolorDirective } from './directives/inputcolor.directive';
-import { EcoNewsComponent } from './component/general/eco-news/eco-news.component';
-import { CreateNewsComponent } from './component/general/eco-news/create-news/create-news.component';
-import { NewsListComponent } from './component/general/eco-news/news-list/news-list.component';
-import { NewsListListViewComponent } from './component/general/eco-news/news-list/news-list-list-view/news-list-list-view.component';
+import { EcoNewsComponent } from './component/eco-news/eco-news.component';
+import { CreateNewsComponent } from './component/eco-news/create-news/create-news.component';
+import { NewsListComponent } from './component/eco-news/news-list/news-list.component';
+import { NewsListListViewComponent } from './component/eco-news/news-list/news-list-list-view/news-list-list-view.component';
 // tslint:disable-next-line: max-line-length
```
Comment (RomanDeveloperAcc) [inline src/app/app.module.ts]: Please, remove commented code

File: src/app/app.module.ts [hunk 2]
```diff
@@ -113,21 +113,22 @@ import { SwiperModule } from 'ngx-swiper-wrapper';
...
-import { EcoNewsDetailComponent } from './component/general/eco-news/eco-news-detail/eco-news-detail.component';
+import { EcoNewsDetailComponent } from './component/eco-news/eco-news-detail/eco-news-detail.component';
 import { CoreModule } from './component/core/core.module';
-import { PostNewsLoaderComponent } from './component/post-news-loader/post-news-loader.component';
-import { EcoNewsWidgetComponent } from './component/general/eco-news/eco-news-detail/eco-news-widget/eco-news-widget.component';
-import { NewsPreviewPageComponent } from './component/general/eco-news/news-preview-page/news-preview-page.component';
+import { PostNewsLoaderComponent } from './component/eco-news/post-news-loader/post-news-loader.component';
+import { EcoNewsWidgetComponent } from './component/eco-news/eco-news-detail/eco-news-widget/eco-news-widget.component';
+import { NewsPreviewPageComponent } from './component/eco-news/news-preview-page/news-preview-page.component';
+import {EcoNewsModule} from './component/eco-news/eco-news.module';
```
Comment (RomanDeveloperAcc) [inline src/app/app.module.ts]: Please add two spaces between module name

File: src/app/component/eco-news/eco-news-routing.module.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { EcoNewsComponent } from './eco-news.component';
+import { CreateNewsComponent } from './create-news/create-news.component';
+import { EcoNewsDetailComponent } from './eco-news-detail/eco-news-detail.component';
+import {NewsPreviewPageComponent} from './news-preview-page/news-preview-page.component';
+import {PostNewsLoaderComponent} from './post-news-loader/post-news-loader.component';
```
Comment (RomanDeveloperAcc) [inline src/app/component/eco-news/eco-news-routing.module.ts]: Please in this line and in the line above add two spaces between module name

File: src/app/app.module.ts (hunk 1 above)
Comment (nvolodymyr) [reply src/app/app.module.ts]: fix

File: src/app/component/eco-news/eco-news-routing.module.ts (hunk 3 above)
Comment (nvolodymyr) [reply src/app/component/eco-news/eco-news-routing.module.ts]: fix

**Pull request merged**